metaseq/cli/train.py

```diff
@@ -274,6 +274,16 @@ def train(
                 metrics.reset_meters("train_inner")
 
         end_of_epoch = not itr.has_next()
+        if end_of_epoch:
```

Comment:
can we leave a comment pointing to this PR or the task and saying we want to delete this after we resolve the issue?

Reply:
Should we merge it, or do you just want to cherry-pick it into a job run? I am fine either way